app/themes/baseTheme.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable quotes */
```

Comment:
I don't see anything in this file that requires disabling the quotes rule.
The strings surrounded by backticks should be converted to single quotes, and then this file should pass linting.

Reply:
Got it -- I had thought that the backpacks were required for styled-components, but looking at the rest of this file I agree that converting to single quotes and removing the `eslint-disable` is the way to go.